portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,32 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import com.google.gson.*;
```

Comment:
Please avoid importing *.

Also, leave an empty line between package and import.

Reply:
Done.